pkg/api/controller.go

```diff
@@ -12,20 +12,27 @@ import (
 	"github.com/anuvu/zot/errors"
 	"github.com/anuvu/zot/pkg/log"
 	"github.com/anuvu/zot/pkg/storage"
+	"github.com/boltdb/bolt"
 	"github.com/gorilla/handlers"
 	"github.com/gorilla/mux"
 )
 
+const (
+	DBFilePath = "./data/db/ZotSearch.db"
+)
+
 type Controller struct {
 	Config     *Config
 	Router     *mux.Router
 	ImageStore *storage.ImageStore
 	Log        log.Logger
 	Server     *http.Server
+	DBPath     string
```

Comment:
I recommend this to go to separate process of its own. In real deployment we can front this with a reverse proxy.

Reply:
Yes, as Ram suggested below